### PR TITLE
Fix 401 403 issue and add tests

### PR DESCRIPTION
**Pull request by johnwiseheart**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -7,13 +7,29 @@ const POST_FILE = 'POSTFILE'
...
 const contentType = {
   GET: 'application/x-www-form-urlencoded',
   POST: 'application/json',
   PUT: 'application/json',
   DELETE: 'application/json',
 }
 
+const networkError = (status, errorType, body, notify, message) => {
+  const errorMessage = message || 'A network error occurred. Please try again.'
+  const e = new Error(errorMessage)
```
Comment (nickw444) [inline src/index.js]: Should we use different Error type instances instead of a string constants?

File: package.json [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
   "name": "twopi-network-utils",
-  "version": "1.0.5",
+  "version": "1.1.0",
   "description": "A set of network utilities to improve code reusability across javascript projects.",
   "main": "./dist/index.js",
   "scripts": {
     "build": "babel src --out-dir dist",
-    "prepublish": "npm run build"
+    "prepublish": "npm run build",
+    "test": "mocha --require babel-core/register src/**/*test.js"
```
Comment (nickw444) [inline package.json]: Can we use eslint too?

File: src/index.js (hunk 1 above)
Comment (johnwiseheart) [inline src/index.js]: as in have an actual JSONParseError instead of just returning an error with that code?

File: package.json (hunk 2 above)
Comment (johnwiseheart) [inline package.json]: ya ill add it

File: src/index.js (hunk 1 above)
Comment (nickw444) [inline src/index.js]: (can be done though) :P

File: src/index.js [hunk 3]
```diff
@@ -7,6 +7,30 @@ const POST_FILE = 'POSTFILE'
 const PUT = 'PUT'
 const DELETE = 'DELETE'
 
+
+// const JSONParseError = 'JSONParseError'
```
Comment (johnwiseheart) [inline src/index.js]: rip one sec

**Comment by nickw444** [comment]
Merge this then i'll setup travis

**Pull request merged**